src/actions/main.js

```diff
@@ -1,82 +0,0 @@
-export const CHECK_LOCATION = "CHECK_LOCATION"
-export const checkLocation = () => {
-    return (dispatch) => {
-        if ("geolocation" in navigator) {
-            navigator
-                .geolocation
-                .getCurrentPosition((location) => {
-                    dispatch(setLocation(location.coords.latitude, location.coords.longitude))
-                    dispatch(getWeather(location.coords.latitude, location.coords.longitude))
-                })
-        } else {
-            return {type: SET_LOCATION, location: null}
-        }
-    }
-}
-
-export const SET_LOCATION = "SET_LOCATION"
-export const setLocation = (lat, lon) => {
-    return {type: SET_LOCATION, location: {lat,lon}}
-}
-
-export const WEATHER_RESULTS = "WEATER_RESULTS"
-export const weatherResults = (weather) => {
-    return {type: WEATHER_RESULTS, weather}
-}
-
-export const GET_WEATHER = "GET_WEATHER"
-export const getWeather = (lat, lon, query) => {
```

Comment:
Yes there is. I prepared the webpack setup to support html5APIfallback more efficiently than the default behavior that webpack dev server brings. With this node setup we can attach a front end html5 router to this project with no problems.